backend/job/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from account.models import KanbanColumn, User
+
+# class Deadline(models.Model):
+#     title = models.CharField(max_length=60)
+#     date = models.DateField(verbose_name="Deadline Date")
+
+#     def to_dict(self):
+#         return {
+#             "id": self.id,
+#             "title": self.title,
+#             "date": self.date,
+#         }
+
+
+class Job(models.Model):
+    # column_id commented as causing tests to fail
+    # column_id = models.ForeignKey(KanbanColumn, on_delete=models.CASCADE)
```

Comment:
This should be uncommented - It was only commented in the first place as I didn't want to troubleshoot the failing tests at midnight

Reply:
leaving commented until the direct tests can pass with just User as a Foreign Key